IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_FastenerRailWeld/DocPropertySet.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_FastenerRailWeld" UniqueId="27eae412-12dc-d9f9-29a1-1578ad14fd41" ApplicableType="IfcFastener/WELD" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
+	<Properties>
+		<DocProperty xsi:nil="true" href="BelongingPanel_2I_S0OrPP7Wvu0OPlABgP" />
+		<DocProperty xsi:nil="true" href="ConstructionCompany_3q5el6dSLCw8hGDPwFONxo" />
+		<DocProperty xsi:nil="true" href="IsWeldedInFactory_0aqg9_hP5xeWnJrydF220" />
+		<DocProperty xsi:nil="true" href="IsLiftingBracket_0Py8wuJJrAwPXDvufAW9rN" />
+		<DocProperty xsi:nil="true" href="RelativePosition_06BnzJB6b9zhMKCUAtPkfJ" />
+		<DocProperty xsi:nil="true" href="TemperatureDuringInstallation_3vvVR5txv9mvLBcLLN7XJV" />
+		<DocProperty xsi:nil="true" href="TrackEdgeId_3LYQ8lEh97fw__L8ZChIgF" />
```

Comment:
Same discussion as usual regarding TrackEdgeId.

Reply:
I have erased it. adb5fdd751ea322bf0649365b080a81b2a458a62